src/utilities/inputProcessors.ts

```diff
@@ -149,6 +150,22 @@ export const processInputReleaseBranches = (): BranchSpec[] | undefined => {
   return validateInputReleaseBranches(parsedInput);
 };
 
+export const processInputConfigFile = (): string | undefined => {
+  const file = parseFileList(getInput(InputParameters.ConfigFile)).shift();
+
+  if (file === undefined) {
+    return;
+  }
+
+  const fileRegex = /\.(?:ya?ml)$/u;
+
+  if (fileRegex.exec(file) === null) {
+    throw new Error('Config file should be a YAML file');
+  }
```

Comment:
Some of the plugins used by semantic-release require function implementations as a properties. Would using a `ts` / `js` file to provide this configuration provide us with more flexibility in the implementation?

Reply:
I discussed this with @merlinnot. We opted for YAML as it is what we currently use across other repositories. We could more options later if desired.